deploy/aws/app/kube/provider.tf

```diff
@@ -21,12 +21,12 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.20.0"
+      version = ">= 3.37.0"
```

Comment:
We probably want to be on 5.x?

Reply:
Current priority was getting this working as is. stacks-dotnet was using 3.20.0 and stacks-dotnet-cqrs was using 3.37.0. I'll add a task to upgrade providers